rust/lance/src/io/writer.rs

```diff
@@ -110,14 +110,18 @@ pub async fn write_manifest(
 pub struct FileWriter {
     object_writer: ObjectWriter,
     schema: Schema,
-    batch_id: i32,
+    pub batch_id: i32,
```

Comment:
Maybe this should be an accessor method? Also we should document what this is (is it the last batch we wrote, or the next one?)

Reply:
Fixed.